### DISCUSSION TITLE
Thanks!

### DISCUSSION THREAD

**Discussion by domportera** [deckerst/aves#609] [General]
just wanted to drop by and say thank you for making this. it's the best android gallery I've come across by a long shot, including OEM ones :)

**Comment by deckerst** [comment]
Thank you for the kind words. I'm glad you enjoy the app.

**Comment by arthuragone** [comment]
Didn't want to create another topic, but as this is not as frequently said, thank you for this app and the time spent. It's one of the 5 most useful on my phone. I've started a small annual support to keep you up.
